### PR TITLE
fixing replies and adding new app icons.

### PR DESCRIPTION
**Pull request by Benjamindodgson**
New app icons and some fixes to replies

### CODE REVIEW

File: iOS/Flows/Conversation/Cells/MessageFooterView.swift [hunk 1]
```diff
@@ -12,7 +12,7 @@ import Combine
 
 class MessageFooterView: BaseView {
     
-    static let height: CGFloat = 70
+    static let height: CGFloat = 86
```
Comment (Benjamindodgson) [inline iOS/Flows/Conversation/Cells/MessageFooterView.swift:15]: Increased size to handle 3 lines of text

File: iOS/UI/Shared/ReplySummaryView.swift [hunk 2]
```diff
@@ -48,32 +48,50 @@ class ReplyView: BaseView {
     let personView = BorderedPersonView()
     let dateLabel = ThemeLabel(font: .xtraSmall)
     let label = ThemeLabel(font: .small)
+    let imageView = UIImageView()
+    
+    static let minimumHeight: CGFloat = 26
```
Comment (Benjamindodgson) [inline iOS/UI/Shared/ReplySummaryView.swift:53]: Supports multiple lines

File: iOS/UI/Shared/ReplySummaryView.swift [hunk 3]
```diff
@@ -82,6 +100,8 @@ class ReplyView: BaseView {
         } else {
             self.label.setText("View reply")
         }
+        
+        self.imageView.image = message.deliveryType.image
```
Comment (Benjamindodgson) [inline iOS/UI/Shared/ReplySummaryView.swift:104]: Added the delivery image

File: iOS/UI/Shared/ReplySummaryView.swift [hunk 4]
```diff
@@ -91,25 +111,10 @@ class ReplyView: BaseView {
...
     
     private var controller: MessageController?
-    
-    var cancellables = Set<AnyCancellable>()
-    
+        
     private let lineDotView = LineDotView()
     private let promptLabel = ThemeLabel(font: .smallBold, textColor: .D1)
     private let promptButton = ThemeButton()
-    private let counter = NumberScrollCounter(value: 0,
```
Comment (Benjamindodgson) [inline iOS/UI/Shared/ReplySummaryView.swift:100]: This was not needed and caused layout issues.

File: iOS/UI/Shared/ReplySummaryView.swift [hunk 5]
```diff
@@ -214,33 +211,10 @@ class ReplySummaryView: BaseView {
...
         
         self.lineDotView.expandToSuperviewHeight()
     }
-
-    private func subscribeToUpdates() {
-        self.cancellables.forEach { cancellable in
-            cancellable.cancel()
-        }
-        
-        self.controller?.repliesChangesPublisher.mainSink { [unowned self] _ in
```
Comment (Benjamindodgson) [inline iOS/UI/Shared/ReplySummaryView.swift:239]: The view is already getting these updates so removing this fixed an extra layout call.

**Review by MartinJibber — APPROVED** [review]

**Pull request merged**